package/libnsh/libnsh.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# libnsh
+#
+################################################################################
+
+LIBNSH_VERSION = 0.4
+LIBNSH_PKGNAME = libnsh
+LIBNSH_SOURCE = $(LIBNSH_PKGNAME)-$(LIBNSH_VERSION).tar.gz
```

Comment:
The _SOURCE is not needed when a package follows std naming rules and is a .tar.gz archive.  Please remove.